desktop/src/App.tsx

```diff
@@ -13,6 +13,8 @@ import { Upload } from "./pages/upload";
 import { ThemeProvider } from "./components/theme";
 import { Media } from "./pages/mediaFiles";
 import { Editor } from "./pages/editor";
+import { VideoGeneratorDummy } from "./pages/videogen";
+import { Layout } from "./components/layout";
```

Comment:
```suggestion
import { Layout } from "./components/layout";
```
I think this can be removed it is already imported

Reply:
yep must have duped in the merge, deleted now